daemon/config.go

```diff
@@ -138,6 +151,8 @@ func (config *Config) InstallCommonFlags(cmd *flag.FlagSet, usageFn func(string)
 	cmd.StringVar(&config.ClusterStore, []string{"-cluster-store"}, "", usageFn("Set the cluster store"))
 	cmd.Var(opts.NewNamedMapOpts("cluster-store-opts", config.ClusterOpts, nil), []string{"-cluster-store-opt"}, usageFn("Set cluster store options"))
 	cmd.StringVar(&config.CorsHeaders, []string{"-api-cors-header"}, "", usageFn("Set CORS headers in the remote API"))
+	cmd.IntVar(&config.MaxDownloadConcurrency, []string{"-max-download-concurrency"}, defaultMaxDownloadConcurrency, usageFn("Set the max concurrent downloads for each pull"))
+	cmd.IntVar(&config.MaxUploadConcurrency, []string{"-max-upload-concurrency"}, defaultMaxUploadConcurrency, usageFn("Set the max concurrent uploads for each push"))
```

Comment:
`-max-download-concurrency` is a long option, and concurrency is a relatively technical term.

What about `-simultaneous-downloads` / `-simultaneous-uploads`? @thaJeztah any ideas?